### ISSUE TITLE
Please write some user documentation

### ISSUE THREAD

**Issue by martinellison** [maks/MGit#421]
The user manual in the wiki needs a lot of more content.

It took me some time to work out how to stage, commit and push some files that I had changed (attn saver: use the toggle drawer menu item, it gives you lots of commands).

Also the README should have a link to the user manual.

**Comment by maks** [comment]
Yes definitely agree! And I'd be happy to receive any contributions for improved documentation  :slightly_smiling_face:
